src/conv-core/isomalloc.C

```diff
@@ -970,6 +973,8 @@ struct isommap
     pup_raw_pointer(p, allocated_extent);
     p | use_rdma;
 
+    p | protect_regions;
+
```

Comment:
maybe add a comment somewhere here about not pupping heap_record and use_recording because they reset after migration

Reply:
I feel like the existing comment by their definitions in the struct would be sufficient.